templates/certificates/dynamic-styles.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Single certificate dynamic styles.
+ * Single certificate .
```

Comment:
```suggestion
 * Single certificate dynamic styles.
```

I don't see any reason to change the comment here.